Dataplaneapi v2.7.3 throws SIGSEGV when starting

When starting the dataplaneapi v2.7.3, following SIGSEGV as below is thrown. With Version 2.7.2 everything works as expected. (Discovered by using latest haproxytech/haproxy-alpine containers)

Command used:
```
/usr/bin/dataplaneapi --host 127.0.0.1 --port 5555 --haproxy-bin /usr/sbin/haproxy --config-file /usr/local/etc/haproxy/haproxy.cfg --reload-cmd "kill -SIGUSR2 1" --restart-cmd "kill -SIGUSR2 1" --reload-delay 5
```

Error:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0x10d9a9b]

goroutine 1 [running]:
github.com/haproxytech/dataplaneapi.SyncWithFileSettings(0xc000195080, 0xc000050130?)
	github.com/haproxytech/dataplaneapi/dataplaneapi_configuration.go:38 +0xfb
main.startServer(0xc000416a00)
	github.com/haproxytech/dataplaneapi/cmd/dataplaneapi/main.go:137 +0x7f2
main.main()
	github.com/haproxytech/dataplaneapi/cmd/dataplaneapi/main.go:59 +0x3d
```

--- a/dataplaneapi_configuration.go
+++ b/dataplaneapi_configuration.go
@@ -35,7 +35,7 @@ func SyncWithFileSettings(server *Server, cfg *configuration.Configuration) { //
 			configStorage.Dataplaneapi.EnabledListeners = &[]string{"http"}
 		}
 	}
-	if configStorage.Dataplaneapi.EnabledListeners != nil && !misc.HasOSArg("", "scheme", "") {
+	if configStorage.Dataplaneapi != nil && configStorage.Dataplaneapi.EnabledListeners != nil && !misc.HasOSArg("", "scheme", "") {
 		server.EnabledListeners = *configStorage.Dataplaneapi.EnabledListeners
 	}
 	if configStorage.Dataplaneapi != nil && configStorage.Dataplaneapi.CleanupTimeout != nil && !misc.HasOSArg("", "cleanup-timeout", "") {

